integration-cli/docker_api_stats_test.go

```diff
@@ -89,7 +90,7 @@ func (s *DockerSuite) TestApiStatsNetworkStats(c *check.C) {
 
 	// Retrieve the container address
 	contIP := findContainerIP(c, id, "bridge")
-	numPings := 4
+	numPings := 1
```

Comment:
nit: Other way would be to use `-i .1`. Not sure what is equivalent for win or why this is set to 4.